strongbox-data-service/src/main/java/org/carlspring/strongbox/gremlin/dsl/EntityTraversalDsl.java

```diff
@@ -112,4 +115,24 @@ public String toString()
                                                                              ((Element) t.get()).property("uuid")
                                                                                                 .orElse("null"))));
     }
+    
+    default <E> GraphTraversal<S, E> property(final String key,
+                                              final Set<String> values,
+                                              GraphTraversal<S, E> graphTraversal)
```

Comment:
@anki2189 `graphTraversal` is superfluous here

Reply:
Then how can we get the same `GraphTraversal` instance on which other properties were set. As per my understanding it is working like chaining of properties

Reply:
> Then how can we get the same GraphTraversal instance on which other properties were set. 

this is `this` instance. you can have a look how other methods implemented here and there is also some [documentation](https://tinkerpop.apache.org/docs/current/reference/#dsl) on this.

Reply:
@sbespalov Can you pls change and push the changes , I am little bit confused.